modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -9,47 +9,73 @@
 [source,yaml]
 ----
 apiVersion: mirror.openshift.io/v1alpha2
 kind: ImageSetConfiguration
-archiveSize: 4
 storageConfig:
- registry:
-   imageURL: example.com/mirror/oc-mirror-metadata
-   skipTLS: false
+  local:
+    path: /home/user/metadata
 mirror:
- platform:
-   architectures:
-     - "s390x"
-   channels:
-     - name: stable-4.13
- operators:
-   - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.13
- helm:
-   repositories:
-     - name: redhat-helm-charts
-       url: https://raw.githubusercontent.com/redhat-developer/redhat-helm-charts/master
-       charts:
-         - name: ibm-mongodb-enterprise-helm
-           version: 0.2.0
- additionalImages:
-   - name: registry.redhat.io/ubi9/ubi:latest
+  platform:
+    channels:
+      - name: stable-4.10
+        minVersion: 4.11.37
+        maxVersion: 4.12.15
+        shortestPath: true
+----
+
+// Moved to second; unchanged
+[discrete]
+[id="oc-mirror-image-set-examples-minimum-to-latest_{context}"]
+== Use case: Including all versions of {product-title} from a minimum to the latest
+
+The following `ImageSetConfiguration` file uses a registry storage backend and includes all {product-title} versions starting at a minimum version of `4.10.10` to the latest version in the channel.
+
+On every invocation of oc-mirror with this image set configuration, the latest release of the `stable-4.10` channel is evaluated, so running oc-mirror at regular intervals ensures that you automatically receive the latest releases of {product-title} images.
+
+.Example `ImageSetConfiguration` file
+[source,yaml]
+----
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+storageConfig:
+  registry:
+    imageURL: example.com/mirror/oc-mirror-metadata
+    skipTLS: false
+mirror:
+  platform:
+    channels:
+      - name: stable-4.10
+        minVersion: 4.10.10
 ----
 
+// Updated:
+// - Added a note below about the maxVersion
+// - Added a note about not necessarily getting all versions in the range
 [discrete]
 [id="oc-mirror-image-set-examples-operator-versions_{context}"]
 == Use case: Including Operator versions from a minimum to the latest
 
 The following `ImageSetConfiguration` file uses a local storage backend and includes only the Red Hat Advanced Cluster Security for Kubernetes Operator, versions starting at 3.68.0 and later in the `latest` channel.
 
+[NOTE]
+====
+When you specify a minimum or maximum version range, you might not receive all Operator versions in that range.
+
+By default, oc-mirror excludes any versions that are marked skipped or replaced in the Operator Lifecycle Manager (OLM) specification. Operator versions that are skipped might be affected by a CVE or contain bugs and a newer version should be used instead. For more information on skipped and replaced versions, see link:https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/[Creating an update graph with OLM].
```

Comment:
```suggestion
By default, oc-mirror excludes any versions that are marked "skipped" or "replaced" in the Operator Lifecycle Manager (OLM) specification. Operator versions that are skipped might be affected by a CVE or contain bugs. Use a newer version instead. For more information on skipped and replaced versions, see link:https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/[Creating an update graph with OLM].
```

Reply:
Not sure if quotation marks or backticks would be more appropriate but I feel setting off "skipped" and "replaced" would help readability.

Reply:
Hmm yeah this one is tough because technically it's not actually the skipped/replaced versions that are marked, it's a newer version that is marked `replaces` or `skips` or `skipRange`. Maybe I'll say something like "....any versions that are skipped or replaced by a newer version..." since it's not really true that the old version is marked. @sherine-k let me know if you have any objections to this updated wording